src/zenml/cli/__init__.py

```diff
@@ -169,7 +176,8 @@
 
 .. code:: bash
 
-   zenml metadata-store register METADATA_STORE_NAME METADATA_STORE_TYPE [--OPTIONS]
+   zenml metadata-store register METADATA_STORE_NAME --type METADATA_STORE_TYPE
+   [--OPTIONS]
```

Comment:
Hmm im wondering whether there should not be a new line here and down below? Might get rendered incorrectly and users might find it hard to copy paste

Reply:
I only made that change because I saw that there was a lot of that that took place in the cleanup. Happy to add in that fix, though.